src/MainWindow.cpp

```diff
@@ -157,13 +159,17 @@ void MainWindow::MessageReceived(BMessage *message)
 			break;
 		case IE_MAINWINDOW_MAINMENU_FILE_ABOUT_SLAYER___:    // "About…" is selected from menu…
 		{
-			BWindow *about = slayer->FindWindow("About Slayer");
-			if (!about)
-				new AboutWindow();
-			else if (about->Lock()) {
-				about->Activate(true);
-				about->Unlock();
-			}
+			BAboutWindow* fAboutWin = new BAboutWindow(B_TRANSLATE_SYSTEM_NAME("Slayer"), SLAYER_VERSION);
```

Comment:
The 2nd parameter is the apps signature, not SLAYER_VERSION.
Would be good to keep the signature one constant that'd be used throughout the code.

Reply:
@humdingerb What would be Slayer's signature? (actually a better question is "what even is a signature?")

Reply:
It's a unique identifier used when creating the BApplication object, see the BeBook on BApplication. It's used by the system e.g. to send BMessages to the correct app.

Reply:
@humdingerb I'm still lost on the signature thing. The BeBook seems to use it as some sort of a directory.

Reply:
Slayer's signature is used here: https://github.com/HaikuArchives/Slayer/blob/master/src/SlayerApp.cpp#L27
Best used as a constant, not copied at several places.

Reply:
@humdingerb So that means I would use "slayer_signature", since it's defined as a constant for the signature. Thanks, @humdingerb :)